jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt

```diff
@@ -54,16 +56,18 @@ class LocalLambdaRunConfigurationFactory(configuration: LambdaRunConfiguration)
 class LocalLambdaRunConfiguration(project: Project, factory: ConfigurationFactory) :
     LambdaRunConfigurationBase<LocalLambdaOptions>(project, factory, "SAM CLI"),
     RefactoringListenerProvider {
+
     companion object {
         private val logger = getLogger<LocalLambdaRunConfiguration>()
     }
 
-    override val lambdaOptions = LocalLambdaOptions()
+    override val serializableOptions = LocalLambdaOptions()
```

Comment:
Why was this name changed? It looks like none of the functionality changed, and the class name hasn't changed.

Reply:
parent class isnt tied to lambda

Reply:
it's overriding a val that's specified in a generic `AwsConnectionsRunConfigurationBase` class that is no longer lambda specific.

Reply:
Nvm didn't see the "override"